lib/linguist/languages.yml

```diff
@@ -3917,6 +3917,18 @@ Pike:
   tm_scope: source.pike
   ace_mode: text
   language_id: 287
+PlantUML:
+  type: markup
```

Comment:
hmm. Why data? Plantuml is by definition a diagram markup language, at least that's how I've seen it described.

Reply:
It is more akin to Latex and Markdown than a 'data' format.

Reply:
Files of `programming` and `markup` languages are counted in statistics. I don't think users would like their e.g., Python repositories to become PlantUML repositories because they have a couple large PlantUML files. That's also the reason why XML is a `data` language in Linguist.

Reply:
marked as data for now. It's still a bit strange to me, as I guess I don't see a problem with plantuml being counted in statistics, as it is a sort of markup/code, but if `data` makes more sense we can keep as data.